src/backend/distributed/commands/utility_hook.c

```diff
@@ -240,13 +242,45 @@ multi_ProcessUtility(PlannedStmt *pstmt,
 
 	if (!CitusHasBeenLoaded())
 	{
+		if (IsA(parsetree, CreateRoleStmt))
+		{
+			if (!IsCitusMainDB())
+			{
+				StringInfo mainDBQuery = makeStringInfo();
+				appendStringInfo(mainDBQuery,
+								 "SELECT citus_internal_start_management_transaction('%lu')",
+								 GetCurrentFullTransactionId().value);
+				RunCitusMainDBQuery(mainDBQuery->data);
+				mainDBQuery = makeStringInfo();
+				appendStringInfo(mainDBQuery,
+								 "SELECT execute_command_on_other_nodes('%s')",
+								 queryString);
```

Comment:
Should use quote_literal_cstr
```suggestion
								 "SELECT execute_command_on_other_nodes(%s)",
								 quote_literal_cstr(queryString));
```